plugin/src/main/java/io/snyk/languageserver/protocolextension/SnykExtendedLanguageClient.java

```diff
@@ -391,6 +380,18 @@ public void snykScan(SnykScanParam param) {
 		setNodeState(param.getStatus(), affectedProductTreeNodes, issueCache);
 	}
 
+	@JsonNotification(value = LsNotificationID.SNYK_FOLDER_CONFIG)
+	public void folderConfig(FolderConfigsParam folderConfigParam) {
+		List<FolderConfig> folderConfigs = folderConfigParam != null ? folderConfigParam.getFolderConfigs() : List.of();
+		CompletableFuture.runAsync(() -> addAll(folderConfigs));
+	}
+
+	public void addAll(List<FolderConfig> folderConfigs) {
```

Comment:
This probably should be moved to a different file, maybe to the to be renamed PreferenceState thingy?